src/controllers/balance-controller.js

```diff
@@ -0,0 +1,62 @@
+import UserBalance from '../models/UserBalance';
+
+export default {
+  checkUserBalance, // eslint-disable-line
+  getBalanceList, // eslint-disable-line
+  updateUserBalance, // eslint-disable-line
+};
+const findError = {
+  message: 'user not found',
+};
+function checkUserBalance(username) {
+  return UserBalance.findOne({ username })
+    .then((user) => {
+      if (user) {
+        return user.balance;
+      }
+      const userBalance = new UserBalance({
+        username,
+        firstName: 'firstName', // Name and Surname using LDAP
+        lastName: 'lastName',
+        balance: 0,
+      });
+      userBalance.save();
+      return 0;
+    });
+}
+function getBalanceList() {
+  const BalanceList = [];
+  return UserBalance.find()
+    .then((users) => {
+      users.forEach((user) => {
+        BalanceList.push({
```

Comment:
Why do we create a new array of objects, can't we simply return whatever UserBalance.find returns?